coalib/parsing/ConfParser.py

```diff
@@ -32,6 +32,23 @@ def __init__(self,
         self.__rand_helper = None
         self.__init_sections()
 
+    def check_valid_lines(self, lines):
+        """
+        This function checks for some invalid config file commands like
+        default_action: *: ShowPatchAction
+        which show error in the previous line. This function gives the
+        correct error and raises System Exit in this case.
+        """
+        for line in lines:
+            occ = line.find(':')
+            line_substr = line[:occ]
+            occ_equals = line_substr.find('=')
+            occ_comment = line_substr.find('#')
+            if occ != -1 and not ((occ_equals != -1 and occ_equals < occ) or
+                                  (occ_comment != -1 and occ_comment < occ)):
+                logging.error('Invalid command "{}" in config file'.format
+                              (line.strip('\n')))
```

Comment:
whee... this looks so hacky... Although our CLI code is in a crappy state, this patch is sailing near the wind... this is not how this should work optimally :/ Isn't there any way directly modifying the parsing routines? Without using a ton of regexes? 😅 

PS: @Makman2 has learned new English: "sailing near the wind"

Reply:
But at least the patch isn't complicated...